### DISCUSSION TITLE
How to allow anonymous page creation / editing?

### DISCUSSION THREAD

**Discussion by ramshankerji** [requarks/wiki#6699] [Help / Questions] [answered]
I am trying to create a wiki for use inside the company.  Since all users shall be on company devices and company network, I planned to remove any authentication requirement inside the wiki software. Similar to Wikipedia. Anyone can create/edit any page. No fear of spam since it shall be only internally accessible. 

However, from the documentation ( https://docs.requarks.io/groups ) it is not clear how to do so. Anything other than Read-access is disabled for "Guest" group. 2nd Approach:  I created another group with all write permissions, but it wouldn't allow the "Guest" user to be added to it.

So now I am lost.

**Comment by NGPixel** [comment, marked as answer]
You must be logged in to edit / create pages.
